TopDownRPG.uproject

```diff
@@ -1,6 +1,6 @@
 {
 	"FileVersion": 3,
-	"EngineAssociation": "5.2",
+	"EngineAssociation": "{0F90F78A-4F3D-EDD1-45C0-01BD5C9145A0}",
```

Comment:
I'm not familiar with the quirks of Unreal, but when trying to load the project i was getting an error with the wrong engine version. Develop branch seems to be on 5.2 and working fine

After forcing this branch it worked perfectly fine